docs/releases/updating_ez_platform.md

```diff
@@ -261,7 +261,7 @@ Some versions require updates to the database. Look through [the list of databas
 
     Since v1.8 you can add a File field to the Form block on a Landing Page. Files uploaded through such a form will be automatically placed in a specific folder in the repository.
 
-    If you are upgrading to v1.8 you need to create this folder and assign it to a new specific Section. Then, add them in the config (for example, in `app/config/default_parameters.yml`, depending on how your configuration is set up):
+    If you are upgrading to v1.8 you need to create this folder and assign it to a new specific Section. Then, add them in the config (for example, in `app/config/default_parameters.yaml`, depending on how your configuration is set up):
```

Comment:
Looks like this section is related to 1.8 which should not be affected by Symfony 4 related changes.